bindings/cs/MongoDB.Libmongocrypt/MongoDB.Libmongocrypt.csproj

```diff
@@ -28,28 +28,26 @@
 
   <PropertyGroup>
     <LibMongoCryptBinaries>../../../cmake-build/$(Configuration)</LibMongoCryptBinaries>
-    <LibMongoCryptBinaries Condition="'$(Configuration)'=='Debug' And '$(OS)' != 'Windows_NT'">../../../debug/</LibMongoCryptBinaries>
```

Comment:
I've verified that building locally in debug mode still results in the proper binaries being copied over (on Windows)

Reply:
IIUC by default LibMongoCryptBinaries refers to `../../../cmake-build/$(Configuration)` unless Configuration is Release, then it refers to `../../../cmake-build/RelWithDebInfo`? And I assume that's needed because there's no notion of a RelWithDebInfo configuration?